spec/file-spec.coffee

```diff
@@ -3,6 +3,7 @@ fs = require 'fs-plus'
 temp = require 'temp'
 File = require '../lib/file'
 PathWatcher = require '../lib/main'
+Q = require 'q'
```

Comment:
It's node modules too; in fact, on the master branch of this repo (and for versions >= 5), we're no longer using Q. Once we upgrade electron, we will be able to stop using Q in Atom, and upgrade it to use a newer pathwatcher version. This PR is against an older version though, in which we haven't yet removed Q.